submissions/krylenger/memory-pair-game/style.css

```diff
@@ -0,0 +1,293 @@
+    height: 30px;
+    padding: 10px 0px;
+    font-size: 30px;
+    font-weight: 500;
+    padding-left: 150px;
+}
+
+.header__title--subtitle {
+    background-color: yellow;
+    font-size: 27px;
+}
+
+.main {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    margin-top: 30px;
+}
+
+.main__grid-container {
+    display: grid;
+    grid-template: repeat(4, 150px) / repeat(5, 150px);
+    grid-gap: 10px;
+    padding: 20px 40px;
+}
+
+.card {
+    perspective: 1000px;
+    font-size: 20px;
+}
+
+.card--0 {
+    grid-column: 1;
+    grid-row: 4;
+
+}
+
+.card--1 {
+    grid-column: 2;
+    grid-row: 3;
+}
+
+.card--2 {
+    grid-column: 2;
+    grid-row: 2;
+}
+
+.card--3 {
+    grid-column: 2;
+    grid-row: 1;
+}
+
+.card--4 {
+    grid-column: 3;
+    grid-row: 1;
+}
+
+.card--5 {
+    grid-column: 4;
+    grid-row: 1;
+}
+
+.card--6 {
+    grid-column: 5;
+    grid-row: 1;
+}
+
+.card--7 {
+    grid-column: 5;
+    grid-row: 2;
+}
+
+.card--8 {
+    grid-column: 5;
+    grid-row: 3;
+}
+
+.card--9 {
+    grid-column: 4;
+    grid-row: 4;
+}
+
+.card--background-green {
+    background-color: palegreen;
+}
+
+.card--orel-img {
+    background-image: url('images/orel.png');
+    background-size: cover;
+}
+
+.card--gopgop-img {
+    background-image: url('images/gopgop.png');
+    background-size: cover;
+}
+
+.card--cadilac-img {
+    background-image: url('images/cadilac.png');
+    background-size: cover;
+}
+
+.card--tanci-img {
+    background-image: url('images/tanci.png');
+    background-size: cover;
+}
+
+.card--dim-img {
+    background-image: url('images/dim.png');
+    background-size: cover;
+}
+
+.card--music-img {
+    background-image: url('images/music.png');
+    background-size: cover;
+}
+
+.card--clef-img {
+    background-image: url('images/clef.png');
+    background-size: cover;
+}
+
+.card--circle-img {
+    background-image: url('images/black-circle.png');
+    background-size: cover;
+}
+
+.card--vertical-line-img {
+    background-image: url('images/vertical-line.png');
+    background-size: cover;
+}
+
+.card--horizontal-line-img {
+    background-image: url('images/horizontal-line.png');
+    background-size: cover;
+}
+
+.card--left-corner-img {
+    background-image: url('images/left-corner.png');
+    background-size: cover;
+}
+
+.card--right-corner-img {
+    background-image: url('images/right-corner.png');
+    background-size: cover;
+}
+
+.card__flipper {
+    -webkit-transition: -webkit-transform 500ms ease;
+    transition: 0.5s;
+    -webkit-transform-style: preserve-3d;
+    transform-style: preserve-3d;
+    position: relative;
+}
+
+.card__flipper--flip {
+    -webkit-transform: rotateY(180deg);
+    transform: rotateY(180deg);
+}
+
+.card__flipper-front, .card__flipper-back {
+    height: 150px;
+    width: 150px;
+    position: absolute;
+    -webkit-backface-visibility: hidden;
+    backface-visibility: hidden;
+    border-radius: 10px;
+}
+
+.card__flipper-front {
```

Comment:
Please use `cursor: pointer;`. it will be better for the UX

Reply:
✅ Done